sdcm/utils/cloud_monitor/report.py

```diff
@@ -170,3 +170,9 @@ def _is_older_than_5days(create_time):
     @staticmethod
     def _is_older_than_7days(create_time):
         return create_time < pytz.utc.localize(datetime.utcnow() - timedelta(days=7))
+
+    def _is_user_be_skipped(self, instance):
+        return instance.owner == "qa" or instance.owner not in self.qa_users
```

Comment:
Maybe it's worthy to do non case-sensitive check

Reply:
It is done in the CloudInstance class
![image](https://user-images.githubusercontent.com/5269241/121519852-df485a80-c9fa-11eb-81f2-3e9a0e5b1761.png)